frontend/src/components/Dialogs/Coordinator.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { useContext, useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { isMobile } from 'react-device-detect';
```

Comment:
The `/frontend ` folder is platform agnostic and we have a more specific way to define were it's being run (web/mobile/desktop/self-hosting).

For this purpose, better if you do `client === 'mobile'` ([example](https://github.com/RoboSats/robosats/blob/054093dea9bb78b54e78134a0244c883bbd3977f/frontend/src/App.tsx#L29))

Reply:
@KoalaSat, can you please check again?